junit-jupiter-params/src/main/java/org/junit/jupiter/params/provider/CsvFileArgumentsProvider.java

```diff
@@ -105,7 +105,7 @@ public boolean hasNext() {
 
 		@Override
 		public Arguments next() {
-			Arguments result = arguments(this.nextCsvRecord);
+			Arguments result = of(this.nextCsvRecord);
```

Comment:
Here, I like to see the "long" form: `Arguments.of(this.nextCsvRecord)`

`of` is really (too) short. Even when "result" is an `Arguments` type.

What happens, when the next line reads (in Java 9):
```
  List<String> list = of("1", "2"); // import static java.util.List.of;
```